### ISSUE TITLE
Invalid slice pitch.

### ISSUE THREAD

**Issue by JAMESBACONPDM** [DurtyFree/durty-cloth-tool#330]
I have the following error:
```
System.Exception: Invalid slice pitch.
   at DurtyClothTool.App.Common.DdsUtils.GetDDSFile(Texture  )
   at GGcObM0mbblFR82rnOw.elDrpT0JuxNUhCmkWJY.qg6pmtKyZ1vCcWW6Ombq(Object )
   at GGcObM0mbblFR82rnOw.elDrpT0JuxNUhCmkWJY.fd10wkWaZG(String  , TextureDictionary  )
   at GGcObM0mbblFR82rnOw.elDrpT0JuxNUhCmkWJY.PY80PZnhbs(String  , String  )
   at DurtyClothTool.App.ViewModels.Commands.AppearanceItem.Clothes.ClothTexture.Export.ExportClothTextureAsDdsCommand.cxrjDLJjl3(List`1  )
   at DurtyClothTool.App.ViewModels.Commands.AppearanceItem.Clothes.ClothTexture.Export.ExportClothTextureAsDdsCommand.Execute(Object  )
   at Telerik.Windows.Controls.RadMenuItem.ExecuteCommand()
   at Telerik.Windows.Controls.RadMenuItem.OnClickImpl()
   at Telerik.Windows.Controls.RadMenuItem.OnClick()
   at Telerik.Windows.Controls.RadMenuItem.ClickItem()
   at Telerik.Windows.Controls.RadMenuItem.HandleMouseUp()
   at Telerik.Windows.Controls.RadMenuItem.OnMouseLeftButtonUp(MouseButtonEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.ReRaiseEventAs(DependencyObject sender, RoutedEventArgs args, RoutedEvent newEvent)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

App Version: **3.0.0.7**
Project targets: **AltV**
DLC name: **bcso**
Collection name: ** // **
Project path: **C:\Users\scott\Downloads\DurtyClothTool.App\clothproject_bcso**
Output path: **C:\Users\scott\Downloads\DurtyClothTool.App**
Disable Extra Ymt Creation: **False**
Create Subfolder For Cloth Drawable Types: **False**
Replace settings enabled: **False // DLC: NoDlc**
Save File Version: **0**
```

Cloth files:
`[Please provide cloth files (ydd, ytd), cloth project and any other related files here + describe reproduction steps]`

**Comment by DurtyFree** [comment]
Hi @JAMESBACONPDM can you please provide the texture (YTD) and model (YDD) files that cause this issue?

**Comment by DurtyFree** [comment]
Fixed next update

**Comment by DurtyFree** [comment]
Fixed in v3.1 https://github.com/DurtyFree/durty-cloth-tool/releases/tag/3.1.0.0
